src/modules/musig/musig-spec.mediawiki

```diff
@@ -0,0 +1,102 @@
+<pre>
+  Title: MuSig Key Aggregation
+  Author:
+  Status: Draft
+  License: BSD-2-Clause
+  Created: 2020-01-19
+</pre>
+
+== Introduction ==
+
+=== Abstract ===
+
+This document describes MuSig Key Aggregation in libsecp256k1-zkp.
+
+=== Copyright ===
+
+This document is licensed under the 2-clause BSD license.
+
+=== Motivation ===
+
+== Description ==
+
+=== Design ===
+
+* A function for sorting public keys allows to aggregate keys independent of the (initial) order.
+* The KeyAgg coefficient is computed by hashing the key instead of key index. Otherwise, if the pubkey list gets sorted, the signer needs to translate between key indices pre- and post-sorting.
+* The second unique key in the pubkey list gets the constant KeyAgg coefficient 1 which saves an exponentiation.
```

Comment:
nit: maybe refer to the Appendix in the paper

(ok, yes, we need to expand this anyway...)

Reply:
done